vcpkg.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "cxx-qt",
+  "version": "0.2.0",
+  "description": "Rust bindings for Qt using CXX",
+  "homepage": "https://kdab.github.io/cxx-qt/book/",
+  "license": "MIT OR Apache-2.0",
+  "supports": "!uwp",
+  "features": {
+    "qt5": {
+      "description": "Build with Qt5",
+      "dependencies": [
```

Comment:
On Linux we also have the following do we need these ? or are they included already?

- qml-module-qtquick2
- qml-module-qttest

Reply:
Evidently those are not needed considering the macOS tests pass without them.

Reply:
Right, just curious how there were linking issues to QtQuick at one point, wondered if it was related. Anyway as you say this works enough and we can continue tweaking in further changes.